Document/0x05b-Basic-Security_Testing.md

```diff
@@ -1644,6 +1644,10 @@ $ cd /system/xbin
 $ chmod 755 tcpdump
 ```
 
+In certain production builds, you might encounter an error `mount: '/system' not in /proc/mounts`.
```

Comment:
Hi @abhaynayar and thanks for sending the PR :)
do you have some more details on this? any specific phone or Android version?

Reply:
This worked on my Moto g5s plus running Android 10.

Reply:
Do you think this is an issue since Android 10?

Reply:
Please provide any references you may have. that'd be very helpful :)

Reply:
I don't know whether this is in Android 10 issue or not (I think it is more to do with certain custom ROMs), not able to test due to lack of Android 10 devices. As for reference I only have a stackoverflow [link](https://stackoverflow.com/questions/28009716/how-can-i-remount-my-android-system-as-read-write-in-a-bash-script-using-adb).

Reply:
Thanks for sharing. So the stack overflow links is 5 years old :-) Meaning it's a problem with certain hardware devices it seems, not the android version.